src/module/fetch.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+
+function Apps() {
```

Comment:
- [Optional] You did well implementing this component, however, I think you could have chosen a more descriptive name for it like `Quotes`. The name of a component informs someone going through your code about the responsibility of a particular component, currently, the name of this component is not very descriptive. I highly recommend giving your components descriptive names in future projects.